include/executor/executor.hh

```diff
@@ -29,10 +29,15 @@ public:
 
   template <InstForwardIterator It>
   void execute(It begin, It end, State &state) const {
+    static std::size_t instrCount = 1;
```

Comment:
If you have more than 1 instance of Executor in program, this counter will be invalid